sh_scrapy/log.py

```diff
@@ -69,17 +69,12 @@ def initialize_logging():
 
     # Scrapy specifics
     if 'SCRAPY_JOB' in os.environ:
-        log.msg("Scrapy %s started" % __version__)
-        log.start = _dummy  # ugly but needed to prevent scrapy re-opening the log
+        logger = logging.getLogger(__name__)
+        logger.info("Scrapy %s started" % __version__)
```

Comment:
Scrapy already [logs](https://github.com/scrapy/scrapy/blob/1.0/scrapy/crawler.py#L212) the info, so keeping it leads to duplicating `Scrapy started ..` line, it should be safe just to drop it. The log below is from a test job on with the updated package:
```
1: | 2019-08-02 09:18:48 | INFO | [sh_scrapy.log] Scrapy 1.7.0 started
3: | 2019-08-02 09:18:48 | INFO | [scrapy.utils.log] Scrapy 1.7.0 started (bot: testspiders)
4: | 2019-08-02 09:18:48 | INFO | [scrapy.utils.log] Versions: ...
```